packages/s/sfml/xmake.lua

```diff
@@ -0,0 +1,116 @@
+        end
+    end
+
+    add_configs("shared",     {description = "Uses the shared libraries (requires .dll or .so)", default = true, type = "boolean"})
+    add_configs("graphics",   {description = "Use the graphics module", default = true, type = "boolean"})
+    add_configs("window",     {description = "Use the window module", default = true, type = "boolean"})
+    add_configs("audio",      {description = "Use the audio module", default = true, type = "boolean"})
+    add_configs("network",    {description = "Use the network module", default = true, type = "boolean"})
+    add_configs("main",       {description = "Link to the sfml-main library", default = true, type = "boolean"})
+
+    on_load("windows", "linux", "macosx", "mingw", function (package)
+        -- linux and macosx are only shared
+        -- windows build does not work in static
+        local shared = package:config("shared") or true
+        local e = ""
+        local a = "sfml-"
+        if is_plat("windows", "mingw") then
+            if not shared then
+                e = "-s"
+            end
+            if package:debug() then
+                e = e .. "-d"
+            end
+        end
+        local main_module = a .. "main"
+        if package:debug() then
+            main_module = main_module .. "-d"
+        end
+
+        if package:config("graphics") then
+            package:add("links", a .. "graphics" .. e)
+            if is_plat("windows", "mingw") then
+                package:add("syslinks", "freetype")
+            end
+        end
+        if package:config("window") or package:config("graphics") then
+            package:add("links", a .. "window" .. e)
+            if is_plat("windows", "mingw") then
+                package:add("syslinks", "opengl32", "gdi32")
+            end
+        end
+        if package:config("audio") then
+            package:add("links", a .. "audio" .. e)
+            if is_plat("windows", "mingw") then
+               package:add("syslinks", "openal32", "flac", "vorbisenc", "vorbisfile", "vorbis", "ogg")
+            end
+        end
+        if package:config("network") then
+            package:add("links", a .. "network" .. e)
+            if is_plat("windows", "mingw") then
+                package:add("syslinks", "ws2_32")
+            end
+        end
+        package:add("links", a .. "system" .. e)
+        if is_plat("windows", "mingw") then
+            package:add("syslinks", "winmm")
+            if package:config("main") then
+                package:add("links", main_module)
+            end
+        end
+    end)
+
+    on_install("windows", "linux", "macosx", "mingw", function (package)
+        os.cp("lib/*", package:installdir("lib"), {rootdir = "lib"})
+        os.cp("include/*", package:installdir("include"), {rootdir = "include"})
+        if is_plat("windows", "mingw") then
+            os.cp("bin/*", package:installdir("lib"), {rootdir = "bin"})
```

Comment:
os.cp("bin", package:installdir())

Reply:
I think that's already correct. I want to copy the contents of ```bin``` into ```lib```

Reply:
ok